src/sandstorm/supervisor.c++

```diff
@@ -905,9 +905,22 @@ void SupervisorMain::unshareOuter() {
     KJ_SYSCALL(unshare(CLONE_NEWUSER | CLONE_NEWNS | CLONE_NEWIPC | CLONE_NEWUTS | CLONE_NEWPID));
 
     // Map ourselves as 1000:1000, since it costs nothing to mask the uid and gid.
+    uid_t fakeUid = 1000;
+    gid_t fakeGid = 1000;
+
+    if (devmode) {
+      // "Randomize" the UID and GID in dev mode. This catches app bugs where the app expects the
+      // UID or GID to be always 1000, which is not true of servers that use the privileged sandbox
+      // rather than the userns sandbox. (The "randomization" algorithm here is only meant to
+      // appear random to a human.)
+      time_t now = time(nullptr);
+      fakeUid = now * 4721 % 2000 + 1;
```

Comment:
These are some weird constants, and I'd prefer you pull data from `/dev/urandom` or something that was more defensible, even though I know it's supposed to be "only random to humans". But out of curiosity, what's with the constants?